features/options/prepare/help.feature

```diff
@@ -11,14 +11,13 @@ Feature: osrm-prepare command line options: help
         And stdout should contain "Options:"
         And stdout should contain "--version"
         And stdout should contain "--help"
-        And stdout should contain "--config"
         And stdout should contain "Configuration:"
         And stdout should contain "--profile"
         And stdout should contain "--threads"
         And stdout should contain "--core"
         And stdout should contain "--level-cache"
         And stdout should contain "--segment-speed-file"
-        And stdout should contain 21 lines
+        And stdout should contain 18 lines
```

Comment:
On my system osrm-prepare only has 2 lines for --config, where does the third line come from?